seqr/views/apis/family_api_tests.py

```diff
@@ -5,32 +5,62 @@
 from django.test import TestCase
 from django.urls.base import reverse
 
-from seqr.views.apis.family_api import update_family_pedigree_image, update_family_assigned_analyst, update_family_fields_handler, update_family_analysed_by, edit_families_handler
+from seqr.views.apis.family_api import update_family_pedigree_image, update_family_assigned_analyst, \
+    update_family_fields_handler, update_family_analysed_by, edit_families_handler, delete_families_handler
 from seqr.views.utils.test_utils import _check_login
 
 FAMILY_GUID = 'F000001_1'
 
 PROJECT_GUID = 'R0001_1kg'
 EMPTY_PROJECT_GUID = 'R0002_empty'
 
+FAMILY_ID_FIELD = 'familyId'
+PREVIOUS_FAMILY_ID_FIELD = 'previousFamilyId'
+
+
 class ProjectAPITest(TestCase):
-    fixtures = ['users', '1kg_project', 'reference_data']
-    multi_db = True
+    fixtures = ['users', '1kg_project']
+
     #  TODO test other family api methods
 
     def test_edit_families_handler(self):
         url = reverse(edit_families_handler, args=[PROJECT_GUID])
         _check_login(self, url)
 
-        # send request
-        req_values = {'families': [{'familyGuid': FAMILY_GUID, 'description': 'Test'}]}
-        response = self.client.post(url, content_type='application/json', data=json.dumps(req_values))
+        # send request with a "families" attribute
+        req_values = {
+            'families': [
+                {'familyGuid': FAMILY_GUID, 'description': 'Test'},
+                {PREVIOUS_FAMILY_ID_FIELD: '1', 'description': 'Test'},
+                {FAMILY_ID_FIELD: '13', 'description': 'Test'}
+            ]
+        }
+        response = self.client.post(url, content_type='application/json',
+                                    data=json.dumps(req_values))
         self.assertEqual(response.status_code, 200)
         response_json = response.json()
 
         self.assertListEqual(response_json.keys(), ['familiesByGuid'])
         self.assertEqual(response_json['familiesByGuid'][FAMILY_GUID]['description'], 'Test')
```

Comment:
If you are adding additional test cases to the request, please also test that the response contains the updated data. In general, if you modify the input to a test you should also be modifying the test on the response. You should add tests for the additional updated families here

Reply:
Okay. I will add response checking for the updated data.